perf/benchmark/runner/runner.py

```diff
@@ -106,7 +106,7 @@ def __init__(
         self.extra_labels = extra_labels
         self.run_baseline = baseline
         self.run_serversidecar = serversidecar
-        self.run_clientsidecar = clientsidecar
+        self.run_bothsidecar = clientsidecar
```

Comment:
if you rename clientsidecar to be `bothsidecar`. Then please also update all of those in this runner.py file, also README.mk

Reply:
done.